cmd/minikube/cmd/start.go

```diff
@@ -19,56 +19,83 @@ package cmd
 import (
```

Comment:
This file contains the most important changes - the switch to using 'oc cluster up' and the merge of relevant 'oc cluster up' options.